demos/defender/defender_demo/core_mqtt_config.h

```diff
@@ -1,5 +1,5 @@
 /*
- * AWS IoT Device SDK for Embedded C V202009.00
```

Comment:
Is it going to be capital V or lowercase v?

Reply:
We are going to leave the V as it as the last release.